query-engine/driver-adapters/js/adapter-planetscale/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prisma/adapter-planetscale",
-  "version": "5.4.0",
+  "version": "5.4.1",
```

Comment:
What do we do with driver adapters between Prisma releases before we have the automated process? I need to get a version out to use it in the regression test on the client side, should I release a real `5.4.1` which will be available for users already, or should I make it `5.5.0-pre`?

Reply:
Actually I did release `5.4.1-pre` under the `integration` tag anyway so I can just use that one as well for tests.

So the main question here is whether I should update the version here and release under `latest` or not.

Reply:
provided this fixes a behavior and doesn't break compatibility, (this should be guaranteed by a test run) I think we should release to latest and make the fix available - whether it's part of this same PR or a new one-

Reply:
PlanetScale test succeeded in https://github.com/prisma/prisma/pull/21377